frontend/src/App.vue

```diff
@@ -20,6 +20,7 @@ export default {
 
 <style>
 @import url(buefy/lib/buefy.css);
+@import url('https://api.tiles.mapbox.com/mapbox-gl-js/v0.47.0/mapbox-gl.css');
```

Comment:
No need for cdn here. Mapbox has .css file in package

Reply:
as i know mapbox does not provide css in module https://www.mapbox.com/mapbox-gl-js/api/